build/targets/KoreBuild.Common.targets

```diff
@@ -53,6 +37,28 @@ as they are subject to change.
 
 <!--
 ###################################################################
+Targets: UseVolatileFeed
+
+Replaces $(RepositoryRoot)\NuGet.config feeds.
+###################################################################
+-->
+  <ItemGroup>
+    <UpdateFeeds Include="AspNetCore" Value="$(NUGET_VOLATILE_FEED_AspNetCore)" Condition="'$(NUGET_VOLATILE_FEED_AspNetCore)' != ''" />
```

Comment:
Thanks. I'd prefer that for simplicity. While possible to use GetEnvironmentVariables, the syntax to split a list of Key=Value pairs is hideous in MSBuild.